SIP/sip-50.md

```diff
@@ -0,0 +1,174 @@
+---
+sip: 50
+title: Web Sockets API 
+description: Definition of the Web Sockets API
+author: ohager
+status: Draft
+type: standard
+category: Interface, SRC
+created: 2024-01-10
+---
+
+
+## Abstract
+This document describes the communication interface for lighweight realtime communication via WebSockets for the Node Software.
```

Comment:
This document describes the lightweight real-time communication interface via WebSockets for the Node Software.